### PR TITLE
Add portuguese.json

### PR DESCRIPTION
**Pull request by thiagoeec**
Hi, @tofi86 

I'm sending an additional pt-BR locale.

p.s.: I hope I'm doing this right.

### CODE REVIEW

File: src/main/resources/localization/portuguese.json [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+	"An update (v%1$s, %2$s) for your current installation (v%3$s, %4$s) is beeing downloaded right now..." : "Uma atualização (v%1$s, %2$s) para sua atual instalação (v%3$s, %4$s) está sendo baixada neste instante...",
+	"Please unzip the downloaded update and replace your current version manually!" : "Por favor descompacte a atualização baixada e substitua sua versão atual manualmente!",
+	
+	"This operating system isn't supported yet!": "Este sistema operacional ainda não é suportado!",
+	
+	"choose EPUB file" : "escolha o arquivo EPUB",
+	"validate EPUB" : "validar EPUB",
+	"epubcheck %1$1s" : "epubcheck %1$1s",
+	"Please choose an EPUB file for validation" : "Por favor escolha um arquivo EPUB para validação",
+	"EPUB files (*.epub)" : "Arquivos EPUB (*.epub"),
```
Comment (thiagoeec) [inline src/main/resources/localization/portuguese.json]: "EPUB files (*.epub)" : "Arquivos EPUB (*.epub)",

**Comment by tofi86** [comment]
Hey Thiago,

that's awesome! :-)

I'll add some code to your branch, so you can test it.

**Comment by tofi86** [comment]
I rebased your branch on my new `master` after applying changes to the Localization mechanism to support `xx_YY` locales like `pt_BR`. That's why I force-pushed to the branch.

**Comment by tofi86** [comment]
Added a couple fof changes to EPUB-Checker classes in order to support pt_BR.

@thiagoeec you can check out this branch now and build a version of pagina EPUB-Checker on your own to test your `pt_BR` localization.

Run `mvn -Dmaven.codesign.skip=true clean package` to create an EXE or APP file in the `target/` folder.

**Comment by thiagoeec** [comment]
> Run `mvn -Dmaven.codesign.skip=true clean package` to create an EXE or APP file in the `target/` folder.

Hi, @tofi86  

I tried to build it a couple of days ago and got this error. Now, I tried again and got the same error. I've tried changing the pom.xml, wich let me go a little ahed, but I always get stuck.
This is the log:

`F:\TEMP\EPUBCheck\PAGINA\EPUB-Checker-thiagoeec-patch-1>mvn -Dmaven.codesign.skip=true clean package
[INFO] Scanning for projects...
[WARNING] The project de.paginagmbh:EPUB-Checker:jar:1.8.0-SNAPSHOT uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[INFO]
[INFO] ---------------------< de.paginagmbh:EPUB-Checker >---------------------
[INFO] Building pagina EPUB-Checker 1.8.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ EPUB-Checker ---
[INFO]
[INFO] --- build-helper-maven-plugin:1.12:timestamp-property (timestamp-property) @ EPUB-Checker ---
[WARNING] Using platform locale (pt_BR actually) to format date/time, i.e. build is platform dependent!
[INFO]
[INFO] --- build-helper-maven-plugin:1.12:parse-version (parse-version) @ EPUB-Checker ---
[INFO]
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ EPUB-Checker ---
[INFO] ShortRevision tag detected. The value is '5'.
[INFO] Executing: cmd.exe /X /C "git rev-parse --verify --short=5 HEAD"
[INFO] Working directory: F:\TEMP\EPUBCheck\PAGINA\EPUB-Checker-thiagoeec-patch-1
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.288 s
[INFO] Finished at: 2018-12-11T19:46:44-02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:buildnumber-maven-plugin:1.4:create (default) on project EPUB-Checker: Cannot get the revision information from the scm repository :
[ERROR] Exception while executing SCM command.: Error while executing command. Error while executing process. Cannot run program "git" (in directory "F:\TEMP\EPUBCheck\PAGINA\EPUB-Checker-thiagoeec-patch-1"): CreateProcess error=2, O sistema não pode encontrar o arquivo especificado
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException`

**Comment by tofi86** [comment]
Oh, you need to clone the repository with git in order to run the build pipeline.
It seems you just downloaded your branch as a zip file, right?

If you're unfamiliar with Git, the easiest way is to use a graphical git client like [Github Desktop](https://desktop.github.com/) or [Git Fork](https://git-fork.com).

You can then clone YOUR fork of pagina EPUB-Checker (https://github.com/thiagoeec/EPUB-Checker/) with the app. there should be a "open in Terminal/CMD/Git bash" button/menu item in all those apps, open it and run the maven command there.

If you find it too hard, I can share a pre-built version of pagina EPUB-Checker from the current branch so you can test it without building with git/maven.

**Comment by thiagoeec** [comment]
Hi, Tobias.

Sorry for the delayed answer.
It worked fine with a little exception. The entries below are not being translated.
```
"Visit IDPF/EpubCheck website" : "Visite o site do IDPF/EpubCheck",
"Visit pagina/EPUB-Checker website" : "Visite o site do pagina/EPUB-Checker",
```

**Comment by thiagoeec** [comment]
Sorry for the mess with commits. I tried to undo the 2 commits, to join them in one, but it ended up creating even more commits.

**Comment by tofi86** [comment]
Alright, I merged some changes from my master branch into your PR branch here and now it should be working again.

**Comment by thiagoeec** [comment]
Thanks, @tofi86.

Just tested and it's perfect now!

**Comment by tofi86** [comment]
Merged. Thanks for your contribution! 🎉

**Pull request merged**